docs/manage/data/view.md

```diff
@@ -31,6 +31,6 @@ For example, if you specify `Rover Rental` for location and click **SEARCH**, yo
 
 To export your captured data from the cloud, see [Export Data](../export).
 
-If you have collected image data, you can use that data to [train image classification model](../train-model) within Viam.
+If you have collected image data, you can use that data to [train image classification model](../../ml/train-model) within Viam.
```

Comment:
```suggestion
Cloud image data can be used to [train machine learning models](../../ml/train-model) within Viam.
```
Since this is really about data that was synced to the cloud

Reply:
```suggestion
You can use cloud image data to [train machine learning models](../../ml/train-model) within Viam.
```